wg-infra/policy/npm.md

```diff
@@ -0,0 +1,36 @@
+# NPM Packages
+
+## Maintainer Access
+
+Maintainer access to npm is governed by the [npm access policy](./access/npm.md).
+
+## Package Security
+
+NPM packages that Electron is responsible for must meet the following requirements:
+
+* Packages **must** be published from CI systems only
+* Automatic release credentials **must** be restricted to the default branch
```

Comment:
Does this need to be amended as per the recent node-abi change since we're releasing to a `3-x-y` maintenance branch?